git_theta/utils.py

```diff
@@ -9,6 +9,8 @@
 class EnvVarConstants:
     CHECKPOINT_TYPE: str = "GIT_THETA_CHECKPOINT_TYPE"
     UPDATE_TYPE: str = "GIT_THETA_UPDATE_TYPE"
+    LSH_SIGNATURE_SIZE: str = "LSH_SIGNATURE_SIZE"
```

Comment:
Unlikely anyone is, but we should prefix the actual env var with `GIT_THETA_*` to ensure we don't step on any toes.

Reply:
Fixed.